domains/certificates/Query.API/API.IntegrationTests/Testcontainers/RegistryFixture.cs

```diff
@@ -4,75 +4,117 @@
 using System.Threading.Tasks;
 using DotNet.Testcontainers.Builders;
 using DotNet.Testcontainers.Containers;
+using DotNet.Testcontainers.Images;
+using DotNet.Testcontainers.Networks;
 using ProjectOrigin.HierarchicalDeterministicKeys;
 using ProjectOrigin.HierarchicalDeterministicKeys.Interfaces;
+using Testcontainers.RabbitMq;
 using Xunit;
 
 namespace API.IntegrationTests.Testcontainers;
 
 public class RegistryFixture : IAsyncLifetime
 {
-    private const string registryImage = "ghcr.io/project-origin/registry-server:0.2.2";
-    private const string electricityVerifierImage = "ghcr.io/project-origin/electricity-server:0.2.2";
-    protected const int GrpcPort = 80;
+    private const string registryImage = "ghcr.io/project-origin/registry-server:1.2.2";
+    private const string electricityVerifierImage = "ghcr.io/project-origin/electricity-server:1.1.0";
+    protected const int GrpcPort = 5000;
```

Comment:
Hardcoded?

Reply:
Yes - it's the port the wallet talks REST on